src/strategies/DeltaNeutralLp.sol

```diff
@@ -17,65 +17,81 @@ import {AccessStrategy} from "./AccessStrategy.sol";
 import {IMasterChef} from "src/interfaces/sushiswap/IMasterChef.sol";
 import {SlippageUtils} from "src/libs/SlippageUtils.sol";
 
+struct LenderInfo {
+    ILendingPool pool; // lending pool
+    ERC20 borrow; // borrowing asset
+    AggregatorV3Interface priceFeed; // borrow asset price feed
+}
+
+struct LpInfo {
+    IUniswapV2Router02 router; // lp router
+    IMasterChef masterChef; // gov pool
+    uint256 masterChefPid; // pool id
+    bool useMasterChefV2; // if we are using MasterChef v2
+    ERC20 sushiToken; // sushi token address, received as reward
+}
+
+struct LendingParams {
+    uint256 assetToDepositRatioBps; // asset to deposit for lending
+    uint256 collateralToBorrowRatioBps; // borrow ratio of collateral
+}
+
 contract DeltaNeutralLp is AccessStrategy {
     using SafeTransferLib for ERC20;
     using FixedPointMathLib for uint256;
     using SlippageUtils for uint256;
 
     constructor(
         AffineVault _vault,
-        ILendingPool _lendingPool,
-        ERC20 _borrow,
-        AggregatorV3Interface _borrowFeed,
-        IUniswapV2Router02 _router,
-        IMasterChef _masterChef,
-        uint256 _masterChefPid,
-        bool _useMasterChefV2,
-        ERC20 _sushiToken,
+        LenderInfo memory lender,
+        LpInfo memory lpProvider,
         IUniswapV3Pool _pool,
-        address[] memory strategists
+        address[] memory strategists,
+        LendingParams memory params
     ) AccessStrategy(_vault, strategists) {
         canStartNewPos = true;
 
-        borrow = _borrow;
-        borrowFeed = _borrowFeed;
+        lendingParams = params;
```

Comment:
You don't need to store this as a struct in storage. You can just store to two uints as immutables.

Reply:
This will increase the number of independent variable in the contract, If its not saving gas, we should keep things more compact.

Reply:
This is a gas efficiency issue. Structs can't be immutable and immutables save gas (no loading from memory or storage).